.pylintrc

```diff
@@ -57,6 +57,7 @@ disable=
  redundant-u-string-prefix,  # still have py2 to support
  logging-format-interpolation,
  logging-not-lazy,
+ use-yield-from,  # whatever, new pylint 3.1 thing
```

Comment:
probably should be an own PR with nicer explanation, but I just wanted to silence it here for now

Reply:
The comment could be

> Yield from cannot be used until we require python3.3 or greater

(I'm not sure we'd want to use yield from everywhere possible then but ot will do for now)

Reply:
Thanks, updated and opened https://github.com/oamg/leapp-repository/pull/1187 with this commit only